GraphConsoleAppV3/GraphRequests.cs

```diff
@@ -0,0 +1,1551 @@
+﻿#region
+
+using Microsoft.Azure.ActiveDirectory.GraphClient;
+using Microsoft.Azure.ActiveDirectory.GraphClient.Extensions;
+using System;
+using System.Collections.Generic;
+using System.Data.Services.Client;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+
+#endregion
+
+namespace GraphConsoleAppV3
+{
+    internal class Requests
+    {
+        public static async Task UserMode()
+        {
+            #region Setup Graph client for user
+
+            ActiveDirectoryClient client;
+
+            //*********************************************************************
+            // setup Microsoft Graph client for user...
+            //*********************************************************************
+            try
+            {
+                client = AuthenticationHelper.GetActiveDirectoryClientAsUser();
+            }
+            catch (Exception e)
+            {
+                Program.WriteError("Acquiring a token failed with the following error: {0}",
+                    Program.ExtractErrorMessage(e));
+                //TODO: Implement retry and back-off logic per the guidance given here:http://msdn.microsoft.com/en-us/library/dn168916.aspx
+                Console.ReadKey();
+                return;
+            }
+
+            #endregion
+
+            User newUser = new User();
+            Application newApp = new Application();
+            IDomain newDomain = new Domain();
+            Group newGroup = new Group();
+            ExtensionProperty newExtension = new ExtensionProperty();
+            ServicePrincipal newServicePrincipal = new ServicePrincipal();
+            OAuth2PermissionGrant newPermissionGrant = new OAuth2PermissionGrant();

```

Comment:
Changing to null

---
In reply to: [81246620](https://github.com/Azure-Samples/active-directory-dotnet-graphapi-console/pull/37#discussion_r81246620) [](ancestors = 81246620)